md/building-community-edition-from-source.md

```diff
@@ -1,158 +1,27 @@
+All source code of Bonita BPM solution is available from the [Bonitasoft GitHub organization](https://github.com/bonitasoft). Each component of the solution has a dedicated repository (e.g. [Engine repository](https://github.com/bonitasoft/bonita-engine)).
 
-Bonita BPM Engine and Web (Portal) components are compiled using [Maven](http://maven.apache.org/index.html). Each source directory contains a configuration file, `pom.xml`. If you make any changes to the source file structure, to add, remove, or replace a source, update the POM file accordingly. Bonita BPM Studio components are built using [Tycho](http://eclipse.org/tycho/), which is an overlay of Maven.
+# Prerequisites
 
 To build the product, you need the following:
 
-* Access to the internet
+* Internet connection
 * Git
-* JDK 1.7
-* Apache Maven 3.0.3
+* JDK 1.7 or 1.8
+* Apache Maven 3.0.5 (version is mandatory)
 * NodeJS
-* An available maven repository with `javax.jms:jms:jar 1.1`
-* Bitrock Install Builder 9.5.3 (optional: required only for building Bonita BPM Studio installers)
 
-The following table shows how to build all the product components. Some components rely on components that are built earlier in the sequence, so the build order must be respected. This is table shows the components for 7.0.0\. For later versions, replace "7.0.0" with the relevant version number. Some components, notably connectors have independent versioning so it is normal that some version numbers are different.
+::: info
+We provide build script for Linux only. Therefore it is recommended to use Linux. If you don't have Linux you can either use a virtual machine or port the build script to Mac or Windows.
+:::
 
-Note: this is not a script. However, you could create a script suitable for your build environment using these commands.
-```bash
-echo "# Building project bonita-jboss-h2-mbean / tag 1.1.0"
-git clone --branch 1.1.0 --single-branch git@github.com:bonitasoft/bonita-jboss-h2-mbean.git bonita-jboss-h2-mbean
-mvn clean install -DskipTests=true -f bonita-jboss-h2-mbean/pom.xml
+# Building Bonita BPM
 
-echo "# Building project bonita-tomcat-h2-listener / tag bonita-tomcat-h2-listener-1.0.1"
-git clone --branch bonita-tomcat-h2-listener-1.0.1 --single-branch git@github.com:bonitasoft/bonita-tomcat-h2-listener.git bonita-tomcat-h2-listener
-mvn clean install -DskipTests=true -f bonita-tomcat-h2-listener/pom.xml
+In order to build Bonita BPM you need to use the script provided in [Build Bonita BPM](https://github.com/Bonitasoft-Community/Build-Bonita-BPM) GitHub project. Use the branch of the project that match the version you want to build.
 
-echo "# Building project bonita-tomcat-valve / tag 7.0.55"
-git clone --branch 7.0.55 --single-branch git@github.com:bonitasoft/bonita-tomcat-valve.git bonita-tomcat-valve
-mvn clean install -DskipTests=true -f bonita-tomcat-valve/pom.xml
+Simply run `bash build-script-7.3.2.sh` and provide required information.
```

Comment:
We include the version in the script because providing a new version as parameter can lead to an incoherent build as connector version are independent of solution version.